week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -14,10 +14,35 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 "think" like a computer and practice your knowledge of basic JavaScript.
 */
 
-function sortAges(arr) {
+// function sortAges(arr) { - another way of only keeping numbers within a string
+//   let dataType;
+//   let newArray = [];
+//   for (let i = 0; i < arr.length; i++) {
+//     dataType = typeof (arr[i]);
+//     if (dataType === "number") {
+//       newArray.push(arr[i]);
+//     }
+//   }
 
+function sortAges(arr) { //new arr - typeof - push
+let newArr = [] // only keeps numbers within a string
+for (let index = 0; index < arr.length; index++) {
+  if (typeof arr[index] === "number" ) {
+    newArr.push(arr[index])
+  }
+}
+  let swap; //sort in ascending order
+for(let i = 0; i < newArr.length; i++) {
+  for (let j= i+1; j < newArr.length; j++) {
+    if(newArr[i] > newArr[j]) {
+      swap = newArr[i];
+      newArr[i] = newArr[j];
+      newArr[j] = swap;
+    }
+  }
+}
+return newArr

```

Comment:
Mind your indentation again. Especially for inner loops and conditions, this can become hard to read and easy to make a typo that can take a long time to trace down. 

Other than that, this is spot on! Do you know how would you change the code to make this sort in descending order?